util/topsql/state/state.go

```diff
@@ -52,3 +52,11 @@ type State struct {
 func TopSQLEnabled() bool {
```

Comment:
Seems that usually this package is referenced as `topsqlstate`, so that this can be named as `Enabled`?

Reply:
How about do this refactor in another PR, since this PR already very large.